src/Storage/Storage.Management/help/New-AzStorageBlobInventoryPolicyRule.md

```diff
@@ -33,11 +34,14 @@ The **New-AzStorageBlobInventoryPolicyRule** cmdlet creates a blob inventory pol
 
 ### Example 1: Create blob inventory policy rule objects, then sets blob inventory policy with the rule objects.
 <!-- Skip: Output cannot be splitted from code -->
+
 ```
 PS C:\> $rule1 = New-AzStorageBlobInventoryPolicyRule -Name Test1 -Destination $containerName -Disabled -Format Csv -Schedule Daily -ContainerSchemaField Name,Metadata,PublicAccess,Last-mOdified,LeaseStatus,LeaseState,LeaseDuration,HasImmutabilityPolicy,HasLegalHold -PrefixMatch con1,con2
 
 PS C:\> $rule2 = New-AzStorageBlobInventoryPolicyRule -Name Test2 -Destination $containerName -Format Parquet -Schedule Weekly -IncludeBlobVersion -IncludeSnapshot -BlobType blockBlob,appendBlob -PrefixMatch aaa,bbb `
                 -BlobSchemaField name,Creation-Time,Last-Modified,Content-Length,Content-MD5,BlobType,AccessTier,AccessTierChangeTime,Expiry-Time,hdi_isfolder,Owner,Group,Permissions,Acl,Metadata
+PS C:\> $rule3 = New-AzStorageBlobInventoryPolicyRule -Name Test3 -Destination $containerName -Format Parquet -Schedule Weekly -IncludeBlobVersion -IncludeSnapshot -IncludeDeleted -BlobType blockBlob,appendBlob -PrefixMatch aaa,bbb `
+                 -ExcludePrefix ccc,ddd -BlobSchemaField name, Last-Modified, BlobType,AccessTier,AccessTierChangeTime,Content-Type,Content-CRC64,CopyId,x-ms-blob-sequence-number,TagCount
```

Comment:
for `name, Last-Modified, BlobType,AccessTier`, there should be no space between 2 schemas, or this won't be take as the value of a single parameter.

Reply:
Done